modules/model/shared/src/main/scala/observe/model/Conditions.scala

```diff
@@ -6,18 +6,25 @@ package observe.model
 import cats.Eq
 import cats.derived.*
 import cats.syntax.all.*
+import coulomb.ops.algebra.cats.all.given
+import eu.timepit.refined.cats.given
 import io.circe.Decoder
 import io.circe.Encoder
+import io.circe.refined.*
+import lucuma.core.circe.coulomb.given
 import lucuma.core.enums.SkyBackground
 import lucuma.core.enums.WaterVapor
 import lucuma.core.model.CloudExtinction
 import lucuma.core.model.ImageQuality
+import lucuma.core.refined.given
 import monocle.Focus
 import monocle.Lens
 
+val x = summon[Decoder[CloudExtinction]]
```

Comment:
do you use `x` somewhere?

Reply:
No, I was just testing things, thank you for catching, I'll remove.